prowler/providers/aws/services/eventbridge/eventbridge_bus_cross_account_access/eventbridge_bus_cross_account_access.py

```diff
@@ -0,0 +1,77 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.eventbridge.eventbridge_client import (
+    eventbridge_client,
+)
+
+
+class eventbridge_bus_cross_account_access(Check):
+    def execute(self):
+        findings = []
+        for bus in eventbridge_client.buses.values():
+            report = Check_Report_AWS(self.metadata())
+            report.resource_id = bus.name
+            report.resource_arn = bus.arn
+            report.resource_tags = bus.tags
+            report.region = bus.region
+            report.status = "PASS"
+            report.status_extended = (
+                f"EventBridge event bus {bus.name} does not allow cross-account access."
+            )
+            if bus.policy and "Statement" in bus.policy:
+                cross_account_access = False
+                if isinstance(bus.policy["Statement"], list):
```

Comment:
Please, move this into a function since we are not going to do that never if not, could be a good starting point to do that for this PR.

Reply:
Done